docs/book/src/reference/watching-resources/testdata/external-indexed-field/api.go

```diff
@@ -19,6 +19,7 @@ package external_indexed_field
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
+
```

Comment:
Ideally, we should not touch on these files because they are not part of the scope.
However, I see that these changes are more related to format so that is fine